frontend/device/kindle/device.lua

```diff
@@ -258,25 +258,6 @@ function Kindle:ambientBrightnessLevel()
     return 4
 end
 
---- Makes sure the C BB cannot be used on devices with a 4bpp fb.
-function Kindle:blacklistCBB()
```

Comment:
I was momentarily confused by these removals without apparent additions in return but the relevant device definitions already specify `canUseCBB = no`. ^_^